docs/adr/tilekit.md

```
@@ -0,0 +1,70 @@
+
+- POST /account/create - create a new local account if not there. Need for onboarding
+- GET /account/status - Current status of accounts
+- POST /account/set-nickname - Update nicknam
+
+ATproto apis coming soon...
+
+## Tiles apis
+
+These are internal apis so that Tiles official UIs can communicate with Tiles daemon
+
+#TODO: these will be populated as we develop
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)